### ISSUE TITLE
Questions from docs

### ISSUE THREAD

**Issue by jackiekazil** [PyCon/pyladies-auction#1]
# [Introduction](https://github.com/PyCon/pyladies-auction/blob/master/introduction.md)

* In this [line](https://github.com/PyCon/pyladies-auction/blame/master/introduction.md#L86), what does "(slides?)" refer to? Are there slides or are you suggesting slides for the intro?
* In this [line](https://github.com/PyCon/pyladies-auction/blame/master/introduction.md#L96), what is a "STOP"? (I don't remember ever using that.) 

# [Accounting](https://github.com/PyCon/pyladies-auction/blob/master/accounting.md)

* Are there any local/state laws we should be aware of for 2020?
* Does the amount of $250 or less stand for Raffle Items?

# [Item Management](https://github.com/PyCon/pyladies-auction/blob/master/item-management.md)

* Is there a PyCon 2020 PyLadies Auction folder already? (I have parts, but not a folder.)
* Is there a slideshow folder that I can build off of to have historical stuff?
* What is the artwork folder for? 
* Can you give me access to PyCon 2019 PyLadies Auction folder? 

# [tldr;](https://github.com/PyCon/pyladies-auction/blob/master/tldr.md)

* Labels for tables -- what are these printed on? 
* Raffle tickets -- do we need to order these? 
* Packing materials -- do we need to order anything specific?
* Raffle slips -- Do we need to order these?

**Comment by jackiekazil** [comment]
@dougn -- most of these can you can answer.

**Comment by dougn** [comment]
Intro
----

* (slides?) is if I am reading from paper notes or slides in a slide deck. At PyCon I have always used note cards or printed paper (an overly large stack of cards or printed out sheets.) Visual comedy. 

* STOP. This was added last year due to complaints of being overly pushy. It is a signal the bidder can use to say STOP pressuring me, back off!  Hand up with fingers spread is calling for runner, palm up with fingers closed, universal signal to STOP.

Accounting
----
Need to ask the accountants if there were changes. $250 has been the limit for years but the laws have changed in the past year. Don't know what does and does not apply to us.

Item Management
----

* Auction folder: Don't know, I have not set one up. 
* Slideshow: In last years main folder
* Artwork: Custom artwork for the year. Mashup of the lady geek w/ years pycon artwork
* Access to 2019: you have it already. I will re-share.

tldr;
---
* JackieA handles the in-auction room sponsor cards. The expo hall cards are printed on [Gartner Studio's](https://www.staples.com/gartner-studios-thank-you-notes-ivory/product_519591) tent cards, blue box, from Staples. Ones w/o thank you. Usually white with pearl emboss. But anything will work.
* Raffle Tickets - we should have PLENTY in stock, and they should not be used.
* The packing tape and spare boxes and other stuff from the staff room. It is nice to have [bubble envelopes](https://www.staples.com/Open-End-1-Bubble-Mailers-7-1-8-x-10-3-4-100-Case/product_630758) is a nice to have. There should be some packed up with the auction stuff already. Most items arrived packed in a box. Keep the packing stuff the items came in.
* Raffle slips - I ordered enough for the next 5 years. You are ALL set.

**Comment by jackiekazil** [comment]
RE: Expo hall cards -- You think these would work? https://www.staples.com/Great-Papers-Triple-Embossed-White-Note-Cards-with-Envelopes/product_718018 

& from what I read, that is the only thing that should be added to the shopping list in #2... am I correct?

**Comment by dougn** [comment]
yup

**Comment by jackiekazil** [comment]
Thank you! Thank you! This is all really helpful!
